main.go

```diff
@@ -104,17 +147,17 @@ func main() {
 				Value:   false,
 				Usage:   "just list tracable events",
 			},
-			&cli.BoolFlag{
+			&cli.StringFlag{
 				Name:    "container",
 				Aliases: []string{"c"},
-				Value:   false,
-				Usage:   "trace only containers",
+				Value:   "new",
```

Comment:
Value is indeed the default value, and we don't want to set to "new" here.
I wrote the comment here, as it seemed as the most suitable place - sorry for the confusion.

We need to keep the current default behavior (tracing new processes). What I meant here, is that if someone uses "--container" without providing any additional command (e.g. all/new) - we need to fall back to the default container mode behavior that exists today, which is tracing newly created containers.

I think that you can use "IsSet" for this purpose in addition to checking that the given string is an empty string